app/services/web-development.html

```diff
@@ -105,6 +105,17 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+        <picture class="d-flex justify-content-around">
+        <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
+        <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
+        <img src="../images/services/wordpress-logo.svg" alt="wordpress logo" title="Wordpress" class="img-fluid">
+        <img src="../images/services/gulp-logo.svg" alt="gulp logo" title="Gulp" class="img-fluid">
+      </picture>
+      <picture class="d-flex justify-content-around mt-4">
```

Comment:
Please correct this. One picture tag can have only one image. We add multiple image tags only when we are serving images for different screen sizes.

Ex:
`<picture> <img scr="..." alt="..."> </picture>`

Reply:
Picture tag added to all images in service pages.